diff --git a/.gitignore b/.gitignore
index 64f7c18..46527d7 100644
--- a/.gitignore
+++ b/.gitignore
@@ -335,3 +335,104 @@ ASALocalRun/
 .mfractor/
 
 
+bgfx/include/bgfx/bgfx.h
+bgfx/include/bgfx/defines.h
+bgfx/include/bgfx/embedded_shader.h
+bgfx/include/bgfx/platform.h
+bgfx/include/bgfx/c99/bgfx.h
+bgfx/include/bimg/bimg.h
+bgfx/include/bimg/decode.h
+bgfx/include/bimg/encode.h
+bgfx/include/bx/allocator.h
+bgfx/include/bx/bx.h
+bgfx/include/bx/commandline.h
+bgfx/include/bx/config.h
+bgfx/include/bx/cpu.h
+bgfx/include/bx/debug.h
+bgfx/include/bx/easing.h
+bgfx/include/bx/endian.h
+bgfx/include/bx/error.h
+bgfx/include/bx/file.h
+bgfx/include/bx/filepath.h
+bgfx/include/bx/float4x4_t.h
+bgfx/include/bx/handlealloc.h
+bgfx/include/bx/hash.h
+bgfx/include/bx/macros.h
+bgfx/include/bx/maputil.h
+bgfx/include/bx/math.h
+bgfx/include/bx/mpscqueue.h
+bgfx/include/bx/mutex.h
+bgfx/include/bx/os.h
+bgfx/include/bx/pixelformat.h
+bgfx/include/bx/platform.h
+bgfx/include/bx/process.h
+bgfx/include/bx/readerwriter.h
+bgfx/include/bx/ringbuffer.h
+bgfx/include/bx/rng.h
+bgfx/include/bx/semaphore.h
+bgfx/include/bx/settings.h
+bgfx/include/bx/simd_t.h
+bgfx/include/bx/sort.h
+bgfx/include/bx/spscqueue.h
+bgfx/include/bx/string.h
+bgfx/include/bx/thread.h
+bgfx/include/bx/timer.h
+bgfx/include/bx/uint32_t.h
+bgfx/include/bx/url.h
+bgfx/include/bx/inline/allocator.inl
+bgfx/include/bx/inline/bx.inl
+bgfx/include/bx/inline/cpu.inl
+bgfx/include/bx/inline/easing.inl
+bgfx/include/bx/inline/endian.inl
+bgfx/include/bx/inline/error.inl
+bgfx/include/bx/inline/float4x4_t.inl
+bgfx/include/bx/inline/handlealloc.inl
+bgfx/include/bx/inline/hash.inl
+bgfx/include/bx/inline/math.inl
+bgfx/include/bx/inline/mpscqueue.inl
+bgfx/include/bx/inline/mutex.inl
+bgfx/include/bx/inline/pixelformat.inl
+bgfx/include/bx/inline/readerwriter.inl
+bgfx/include/bx/inline/ringbuffer.inl
+bgfx/include/bx/inline/rng.inl
+bgfx/include/bx/inline/simd_ni.inl
+bgfx/include/bx/inline/simd128_langext.inl
+bgfx/include/bx/inline/simd128_neon.inl
+bgfx/include/bx/inline/simd128_ref.inl
+bgfx/include/bx/inline/simd128_sse.inl
+bgfx/include/bx/inline/simd128_swizzle.inl
+bgfx/include/bx/inline/simd256_avx.inl
+bgfx/include/bx/inline/simd256_ref.inl
+bgfx/include/bx/inline/sort.inl
+bgfx/include/bx/inline/spscqueue.inl
+bgfx/include/bx/inline/string.inl
+bgfx/include/bx/inline/uint32_t.inl
+bgfx/include/compat/freebsd/alloca.h
+bgfx/include/compat/freebsd/dirent.h
+bgfx/include/compat/freebsd/malloc.h
+bgfx/include/compat/freebsd/signal.h
+bgfx/include/compat/ios/malloc.h
+bgfx/include/compat/mingw/alloca.h
+bgfx/include/compat/mingw/dirent.h
+bgfx/include/compat/mingw/sal.h
+bgfx/include/compat/mingw/salieri.h
+bgfx/include/compat/mingw/specstrings_strict.h
+bgfx/include/compat/mingw/specstrings_undef.h
+bgfx/include/compat/msvc/alloca.h
+bgfx/include/compat/msvc/dirent.h
+bgfx/include/compat/msvc/inttypes.h
+bgfx/include/compat/msvc/stdbool.h
+bgfx/include/compat/msvc/pre1600/stdint.h
+bgfx/include/compat/osx/malloc.h
+bgfx/include/tinystl/allocator.h
+bgfx/include/tinystl/buffer.h
+bgfx/include/tinystl/hash_base.h
+bgfx/include/tinystl/hash.h
+bgfx/include/tinystl/LICENSE
+bgfx/include/tinystl/new.h
+bgfx/include/tinystl/stddef.h
+bgfx/include/tinystl/string.h
+bgfx/include/tinystl/traits.h
+bgfx/include/tinystl/unordered_map.h
+bgfx/include/tinystl/unordered_set.h
+bgfx/include/tinystl/vector.h
diff --git a/Directory.Build.targets b/Directory.Build.targets
index efedef3..343b7bb 100644
--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -2,7 +2,7 @@
 
     <ItemDefinitionGroup>
         <ClCompile>
-            <AdditionalIncludeDirectories>$(OpenXrSdkPath)\include;$(SharedPath);$(SharedPath)\ext;$(IntDir)\CompiledShaders;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>
+            <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory)\bgfx\include;$(OpenXrSdkPath)\include;$(SharedPath);$(SharedPath)\ext;$(IntDir)\CompiledShaders;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>
         </ClCompile>
     </ItemDefinitionGroup>
 
diff --git a/samples/SampleSceneWin32/SampleSceneWin32.vcxproj b/samples/SampleSceneWin32/SampleSceneWin32.vcxproj
index dc673b4..c5c3700 100644
--- a/samples/SampleSceneWin32/SampleSceneWin32.vcxproj
+++ b/samples/SampleSceneWin32/SampleSceneWin32.vcxproj
@@ -72,6 +72,7 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
       <AdditionalDependencies>windowsapp.lib;bgfxd.lib;bxd.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalLibraryDirectories>..\..\bgfx\lib\$(Configuration)\$(Platform)</AdditionalLibraryDirectories>
     </Link>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
